zng/error.go

```diff
@@ -64,3 +65,7 @@ func (t *TypeOfError) ZSON() string {
 func (t *TypeOfError) ZSONOf(zv zcode.Bytes) string {
 	return QuotedString(zv, false)
 }
+
+func IsMissing(zv Value) bool {
+	return zv.Type == TypeError && byteconv.UnsafeString(zv.Bytes) == missing
```

Comment:
The compiler is smart enough to avoid allocation here.
```suggestion
	return zv.Type == TypeError && string(zv.Bytes) == missing
```